src/packages/console/src/pages/verifyEmailPage.tsx

```diff
@@ -46,32 +31,39 @@ export const VerifyEmailPage = (): React.ReactElement => {
   });
 
   return (
-    <Container component='main' maxWidth='sm'>
-      <Paper className={classes.paper}>
-        <Typography component='h1' variant='h5'>
+    <ContainingView isCenteredHorizontally>
+      <ResponsiveContainingView size={12} sizeResponsive={{ small: 8, medium: 6, large: 5 }}>
+        <Stack direction={Direction.Vertical} isFullHeight={true}>
+          <Stack.Item growthFactor={1} shrinkFactor={1} />
+          <Box variant='card' isFullWidth={false}>
+            <Text variant='header5' tag='h1' alignment={TextAlignment.Center}>
           Please check your email
```

Comment:
this line should be tabbed in twice more

Reply:
I ran lint-fix and the file got changed.
So should I tab them, then the lint fails

Reply:
you should tab the Please check your email line, lint wont complain